### PR TITLE
Add support for sky.money

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/sky/decoder.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        ):
+            return DEFAULT_DECODING_OUTPUT
+
+        received_address = hex_or_bytes_to_address(context.tx_log.topics[2])
+        if (assets_amount := token_normalized_value_decimals(
+            token_amount=hex_or_bytes_to_int(context.tx_log.data[0:32]),
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )) == ZERO:
+            return DEFAULT_DECODING_OUTPUT
+        shares_amount = token_normalized_value_decimals(
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/sky/decoder.py:133]:
```suggestion
            return DEFAULT_DECODING_OUTPUT

        shares_amount = token_normalized_value_decimals(
```

**Review by LefterisJP — COMMENTED** [review]
just some small questions/comments

File: rotkehlchen/chain/ethereum/modules/sky/decoder.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+        if None in (mkr_event, sky_event):
+            log.error(f'Failed to decode mkr migration at {context.transaction}')
+            return DEFAULT_DECODING_OUTPUT
+
+        maybe_reshuffle_events([mkr_event, sky_event], context.decoded_events)
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_susds_events(self, context: DecoderContext) -> DecodingOutput:
+        if not (
```
[the remark this answers is not in the document]
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/modules/sky/decoder.py:121]: I need the `is_deposit` variable later and that would mean that I would need to repeat the check

File: rotkehlchen/chain/ethereum/modules/makerdao/decoder.py [hunk 3]
```diff
@@ -553,6 +562,28 @@ def decode_saidai_migration(self, context: DecoderContext) -> DecodingOutput:
...
+
+        amount = token_normalized_value_decimals(
+            token_amount=hex_or_bytes_to_int(context.tx_log.data[0:32]),
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )
+        event = self.base.make_event_from_transaction(
+            transaction=context.transaction,
+            tx_log=context.tx_log,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
```
[the remark this answers is not in the document]
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/modules/makerdao/decoder.py:577]: It seems that we only burn gas. This is a really special case so I wouldn't create an event just for this. The sky decoder takes this event and then modifies it to pick the migration so I wouldn't add a new event type

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**